README.md

```diff
@@ -7,17 +7,7 @@
 [![devDependency Status](https://david-dm.org/restify/clients/dev-status.svg)](https://david-dm.org/restify/clients#info=devDependencies)
 [![bitHound Score](https://www.bithound.io/github/restify/clients/badges/score.svg)](https://www.bithound.io/github/restify/clients/master)
 
-> HttpClient, StringClient, and JsonClient extracted from restify
```

Comment:
I don't understand how this information is redundant, do you mind clarifying?

Reply:
It’s just a copy of the next paragraph, **Usage**, thus reading the README from top to bottom is irritating. It’s the same text, just with slightly different formatting.

Reply:
Ah I see it now :) That makes sense, thanks for clarifying!

Reply:
I would even drop:

> The **restify-clients** [Node](https://nodejs.org/) package provides HTTP clients from [restify](https://github.com/restify/node-restify).

Because even though this repository results from the extraction of clients from a previous restify version, someone who would click on the link to node-restify wouldn't see any client code there, unless they looked at older branches, so it seems it'd be more confusing than helpful.

Also, I don't think the fact that clients code was extracted from the original restify repository is very helpful for users coming to this repository.